client/rest-client/src/main/java/org/infinispan/client/rest/impl/okhttp/RestCacheClientOkHttp.java

```diff
@@ -53,12 +53,24 @@ public String name() {
    }
 
    @Override
-   public CompletionStage<RestResponse> synchronizeData() {
+   public CompletionStage<RestResponse> synchronizeData(Integer readBatch, Integer threads) {
       Request.Builder builder = new Request.Builder();
-      builder.url(cacheUrl + "?action=sync-data").get();
+      StringBuilder sb = new StringBuilder(cacheUrl + "?action=sync-data");
+      if (readBatch != null) {
+         sb.append("&read-batch=" + readBatch);
```

Comment:
nitpick: shouldn't the string concatenation be avoided with StringBuilder?
```suggestion
         sb.append("&read-batch=").append(readBatch);
```
same below... and above...